frontend/app/src/locales/fr.json

```diff
@@ -4028,7 +4028,8 @@
     },
     "hint": {
       "description": "* tapez le filtre puis appuyez sur entrée, par exemple.",
-      "example": "start = 26/05/2022"
+      "example": "start = 26/05/2022",
+      "edit_note": "Double-click a filter to edit"
```

Comment:
Nitpick, if you're not gonna translate this in other languages, just leave it in other locale files, and just fill the `en.json`

Reply:
I saw they had some English in them hence